harvard-lts/drs-migration-dashboard
# Pulling data from local json files.

**Pulling data from local json files.**
* * *
# What does this Pull Request do?
This replaces the Google Doc with files in a folder in the container (`/app/static/files`).

# How should this be tested?

A description of what steps someone could take to:
* Take the .zip file Maura C. provided and unzip it in the `/app/static/files` folder. (You really only need the two .json files: `drs_migration.json` and `failed_file.json`)
* Restart the container. You should see the charts show up properly.
* If you view the source, you should see the JSON data it is using in between `<script>` tags on lines 16-19.
* You can update data in the files, and you should see it reflected on the site if you refresh.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No 
- integration tests? No

# Interested parties
@dl-maura @mferrarini @enriquediaz @awoods
